package.json

```diff
@@ -2,6 +2,7 @@
   "devDependencies": {
     "sass": "^1.54.4"
   },
+  "license": "GPL-3.0-only",
```

Comment:
Information on the `license` property: [Link](https://docs.npmjs.com/cli/v9/configuring-npm/package-json#license)

Information on our specific license: [Link](https://opensource.org/licenses/?ls=GNU+GENERAL+PUBLIC+LICENSE)

Our license: `GNU GENERAL PUBLIC LICENSE v.3`